code/procs/jobprocs.dm

```diff
@@ -433,6 +433,8 @@
 			H.traitHolder.removeTrait("pilot")
 			H.traitHolder.removeTrait("sleepy")
 			H.traitHolder.removeTrait("puritan")
+		if (map_setting == "NADIR") //Nadir: pilot trait screws the pilot and adds sub when sub should not otherwise exist.
+			H.traitHolder.removeTrait("pilot")
```

Comment:
Maybe send a message to people so they don't get confused? "You weren't able to fly your pod because it would have melted in this environment!" or something